dub.json

```diff
@@ -10,7 +10,7 @@
 		"StdLoggerDisableWarning"
 	],
 	"dependencies": {
-		"libdparse": "~>0.7.0",
+		"libdparse": "~>0.7.1-beta.2",
```

Comment:
Btw normally the `dub.selections.json` is commited as well. Then there's no need to modify the `dub,json` file ...